### PR TITLE
[DSS-86]:  Storybook: remove mocks from sidebar

### PR DESCRIPTION
**Pull request by pixelflips**
## Description
Removes the `mocks` directory and all related files in order to remove the mocks section from the sidebar.

## Screenshots

|  Before  |  After  |
|--------|--------|
|Screen Shot 2022-10-10 at 2 50 27 PM|Screen Shot 2022-10-10 at 2 55 17 PM|

## Testing in `sage-lib`
- Navigate to [Storybook](http://localhost:4100/)
- Verify Mocks section no longer displays in sidebar

## Testing in `kajabi-products`
1. (**LOW**) Removed Mocks section from Storybook sidebar. Documentation only update, no effect on KP expected.

## Related
https://kajabi.atlassian.net/browse/DSS-86

### CODE REVIEW

**Review by anechol — APPROVED** [review]
LGTM!

File: packages/sage-react/lib/mocks/contact-profile-full/ContactProfileFull.story.jsx [hunk 1]
```diff
@@ -6,11 +6,13 @@ import { Root } from './components';
 // See https://kajabi.atlassian.net/browse/SAGE-361
 import './styles.css';
 
-export default {
-  title: 'Mocks/Contact Profile - Full view',
-  argTypes: {},
-  args: {}
-};
+// export is commented out in order to remove mocks from sidebar
```
Comment (monkeypox8) [inline packages/sage-react/lib/mocks/contact-profile-full/ContactProfileFull.story.jsx:9]: I may be missing some context here, but is the `mocks` directory still needed at all? If mocks are indeed being phased out, we'd be able to cut ~60 files from `sage-react` by removing the directory.

It seems doubtful anyone would navigate directly to the deprecated view(s) when hidden, and the file structure would still exist in the git history should we ever need to get back to it.

**Review by monkeypox8 — APPROVED** [review]
👍 LGTM! Left a question on whether we need to keep the files around.

File: packages/sage-react/lib/mocks/contact-profile-full/ContactProfileFull.story.jsx (hunk 1 above)
Comment (pixelflips) [reply packages/sage-react/lib/mocks/contact-profile-full/ContactProfileFull.story.jsx:9]: @monkeypox8 I am glad you brought that up. I had the same thought but wasn't quite sure so decided to leave the files in place. If it did come up, I planned to bring it up in today's sync. Great callout and I'll bring it up. Thanks again!

**Comment by pixelflips** [comment]
@Kajabi/ux-devs Based on group feedback, I have removed all related files and will request another round of reviews.

File: packages/sage-react/lib/mocks/contact-profile-full/ContactProfileFull.story.jsx (hunk 1 above)
Comment (pixelflips) [reply packages/sage-react/lib/mocks/contact-profile-full/ContactProfileFull.story.jsx]: @monkeypox8 I have removed all files in the mocks directory and will request another quick review. Thanks again for the suggestion

**Review by monkeypox8 — APPROVED** [review]
👍 LGTM!

**Review by ju-Skinner — APPROVED** [review]

**Pull request merged**